src/system/BlocksModule/Container/LinkContainer.php

```diff
@@ -24,10 +24,12 @@ class LinkContainer implements LinkContainerInterface
      * @var Translator
      */
     private $translator;
+
```

Comment:
I guess we need to decide here. I usually leave no space between property declarations. Otherwise we will keep 'fixing' each others code 😉 @cmfcmf @Guite others? opinions?